Dockerfile

```diff
@@ -1,23 +1,18 @@
-FROM node:dubnium-stretch-slim
+FROM alpine:3.11
 
-WORKDIR /home/app
-
-RUN apt update \
-    && apt install -y curl ffmpeg \
-    && rm -rf /var/lib/apt/lists/*
+RUN apk update \
+    && apk add curl ffmpeg nodejs npm python \
+    && rm -rf /var/cache/apk/*
 
 # This is on a separate line because youtube-dl needs to be frequently updated
-RUN apt update \
-    && apt install -y youtube-dl \
-    && rm -rf /var/lib/apt/lists/*
+RUN apk update \
+    && apk add youtube-dl \
+    && rm -rf /var/cache/apk/*
```

Comment:
```suggestion
RUN apk add --no-cache youtube-dl
```

Reply:
With --no-cache it still remains cache in /var/cache/apk/

Reply:
```
$ docker run alpine ash
/ # apk add --no-cache tree
fetch http://dl-cdn.alpinelinux.org/alpine/v3.11/main/x86_64/APKINDEX.tar.gz
fetch http://dl-cdn.alpinelinux.org/alpine/v3.11/community/x86_64/APKINDEX.tar.gz
(1/1) Installing tree (1.8.0-r0)
Executing busybox-1.31.1-r9.trigger
OK: 7 MiB in 19 packages
/ # tree /var/cache/apk/
/var/cache/apk/

0 directories, 0 files
/ #
```
Nope.

Reply:
isn't apk update required? Because you didn't use it there is no cache from update in /var/cache/apk/.